src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -25,12 +27,16 @@
 public class JsonAdaptedPerson {
 
     public static final String MISSING_FIELD_MESSAGE_FORMAT = "Person's %s field is missing!";
-
     private final String name;
     private final String phone;
     private final String email;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+
     private final String address;
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+
```

Comment:
newline shouldnt be in between an annotation and the field,
`@Annotation` \
`private final ...`